app/src/org/commcare/fragments/personalId/PersonalIdPhoneFragment.java

```diff
@@ -236,6 +235,7 @@ private void displayPhoneNumber(String fullPhoneNumber) {
     }
 
     private void onContinueClicked() {
+        FirebaseAnalyticsUtil.reportPersonalIDContinueClicked(PersonalIdPhoneFragment.class.getSimpleName(),null);
```

Comment:
you should instead use `this.getClass().getSimpleName()` so that we make sure that we are referencing the current class only.